iree/compiler/Conversion/LinalgToSPIRV/CodeGenOptionUtils.cpp

```diff
@@ -32,6 +32,12 @@ SPIRVCodegenOptions getSPIRVCodegenOptionsFromClOptions() {
                      "SPIR-V code generation"),
       llvm::cl::ZeroOrMore, llvm::cl::MiscFlags::CommaSeparated);
 
+  static llvm::cl::list<unsigned> clInvocationTileSizes(
+      "iree-spirv-invocation-tile-size",
+      llvm::cl::desc("Set tile sizes to use for the second level tiling for "
```

Comment:
Just say "Set tile sizes for each invocation when tiling Linalg ops"? Second level is really internal implementation (which actually if we count subgroup it's the third level). 

Also we should. rename `iree-spirv-tile-size` to `iree-spirv-workgroup-tile-size` to be clear?

Reply:
Done.